ui/scripts/templates.js

```diff
@@ -1447,7 +1447,11 @@
                                                  label: 'label.action.delete.template',
                                                  messages: {
                                                      confirm: function(args) {
-                                                         return 'message.action.delete.template';
+                                                         if(args.context.templates[0].crossZones == true) {
+                                                             return 'This is a cross zone template and will be deleted from all the zones. Are you sure you want to proceed?';
```

Comment:
@karuturi can you add this to a new dictionary, in messages.properties etc so this can be translated in other languages?